File: Java/Bryann Valderrama/Geometria/gradtoRad_RadtoGrad.java
public class Main {

    static double DegToRad(double d) {
        return d * Math.PI / 180.0;
    }

    static double RadToDeg(double r) {
        return r * 180.0 / Math.PI;
    }

    public static void main(String[] args) {
        System.out.println(DegToRad(360));
        System.out.println(RadToDeg(6.283185307179586));
    }
}

File: Codigos estudiantes por lenguaje/PY/Bryann Valderrama/Geometria/gradtoRad_RadtoGrad.py
import math
from sys import stdout, stdin
wr = stdout.write


def degToRad(d):
    return d * math.pi / 180


def radToDeg(r):
    return r * 180 / math.pi


wr(f'{degToRad(360)}\n')
wr(f'{radToDeg(6.283185307179586)}\n')
